From b8246712825e5c6b6e1916d027debce19cfd83e6 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Tue, 26 Feb 2013 18:21:32 +0200
Subject: ARM: i.MX6: cm-fx6: several minor adjustments

1) remove wrong SD3 pin muxes
2) remove some comment and white space
3) reorder GPIO definitions to be in increasing order

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/board-cm-fx6-dl.h |    8 --------
 arch/arm/mach-mx6/board-cm-fx6-q.h  |    7 -------
 arch/arm/mach-mx6/board-cm-fx6.c    |    3 +--
 3 files changed, 1 insertion(+), 17 deletions(-)

diff --git a/arch/arm/mach-mx6/board-cm-fx6-dl.h b/arch/arm/mach-mx6/board-cm-fx6-dl.h
index 1a537f8..f91839f 100644
--- a/arch/arm/mach-mx6/board-cm-fx6-dl.h
+++ b/arch/arm/mach-mx6/board-cm-fx6-dl.h
@@ -19,7 +19,6 @@
 #include <mach/iomux-mx6dl.h>
 
 static iomux_v3_cfg_t cm_fx6_dl_pads[] = {
-
 	/* UART4 for debug */
 	MX6DL_PAD_KEY_COL0__UART4_TXD,
 	MX6DL_PAD_KEY_ROW0__UART4_RXD,
@@ -65,13 +64,6 @@ static iomux_v3_cfg_t cm_fx6_dl_pads[] = {
 	MX6DL_PAD_SD3_DAT1__USDHC3_DAT1_50MHZ,
 	MX6DL_PAD_SD3_DAT2__USDHC3_DAT2_50MHZ,
 	MX6DL_PAD_SD3_DAT3__USDHC3_DAT3_50MHZ,
-	MX6DL_PAD_SD3_DAT4__USDHC3_DAT4_50MHZ,
-	MX6DL_PAD_SD3_DAT5__USDHC3_DAT5_50MHZ,
-	MX6DL_PAD_SD3_DAT6__USDHC3_DAT6_50MHZ,
-	MX6DL_PAD_SD3_DAT7__USDHC3_DAT7_50MHZ,
-	MX6DL_PAD_SD3_RST__USDHC3_RST,
-	/* SD3 VSelect */
-	MX6DL_PAD_GPIO_18__USDHC3_VSELECT,
 	/* SD3_CD and SD3_WP */
 	MX6DL_PAD_NANDF_CS0__GPIO_6_11,
 	MX6DL_PAD_NANDF_CS1__GPIO_6_14,
diff --git a/arch/arm/mach-mx6/board-cm-fx6-q.h b/arch/arm/mach-mx6/board-cm-fx6-q.h
index 4cf0f7d..c32edb2 100644
--- a/arch/arm/mach-mx6/board-cm-fx6-q.h
+++ b/arch/arm/mach-mx6/board-cm-fx6-q.h
@@ -64,13 +64,6 @@ static iomux_v3_cfg_t cm_fx6_q_common_pads[] = {
 	MX6Q_PAD_SD3_DAT1__USDHC3_DAT1_50MHZ,
 	MX6Q_PAD_SD3_DAT2__USDHC3_DAT2_50MHZ,
 	MX6Q_PAD_SD3_DAT3__USDHC3_DAT3_50MHZ,
-	MX6Q_PAD_SD3_DAT4__USDHC3_DAT4_50MHZ,
-	MX6Q_PAD_SD3_DAT5__USDHC3_DAT5_50MHZ,
-	MX6Q_PAD_SD3_DAT6__USDHC3_DAT6_50MHZ,
-	MX6Q_PAD_SD3_DAT7__USDHC3_DAT7_50MHZ,
-	MX6Q_PAD_SD3_RST__USDHC3_RST,
-	/* SD3 VSelect */
-	MX6Q_PAD_GPIO_18__USDHC3_VSELECT,
 	/* SD3_CD and SD3_WP */
 	MX6Q_PAD_NANDF_CS0__GPIO_6_11,
 	MX6Q_PAD_NANDF_CS1__GPIO_6_14,
diff --git a/arch/arm/mach-mx6/board-cm-fx6.c b/arch/arm/mach-mx6/board-cm-fx6.c
index c5446d2..7ad4961 100644
--- a/arch/arm/mach-mx6/board-cm-fx6.c
+++ b/arch/arm/mach-mx6/board-cm-fx6.c
@@ -79,9 +79,9 @@
 #include "board-cm-fx6-dl.h"
 
 /* GPIO PIN, sort by PORT/BIT */
-#define CM_FX6_GREEN_LED		IMX_GPIO_NR(2, 31)
 #define CM_FX6_TSPENDOWN		IMX_GPIO_NR(2, 15)
 #define CM_FX6_ECSPI1_CS0		IMX_GPIO_NR(2, 30)
+#define CM_FX6_GREEN_LED		IMX_GPIO_NR(2, 31)
 #define CM_FX6_ECSPI1_CS1		IMX_GPIO_NR(3, 19)
 
 #define MX6_ARM2_LDB_BACKLIGHT		IMX_GPIO_NR(1, 9)
@@ -355,7 +355,6 @@ static struct ads7846_platform_data ads7846_config = {
 static struct spi_board_info cm_fx6_spi0_board_info[] = {
 #if defined(CONFIG_MTD_M25P80) || defined(CONFIG_MTD_M25P80_MODULE)
 	{
-		/* The modalias must be the same as spi device driver name */
 		.modalias	= "m25p80",
 		.max_speed_hz	= 20000000,
 		.bus_num	= 0,
-- 
1.7.9.5

